incubating/wrappers/java/src/main/java/io/seldon/wrapper/api/ModelPredictionController.java

```diff
@@ -11,32 +19,79 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @ConditionalOnExpression("${seldon.api.model.enabled:false}")
-public class ModelRestController {
```

Comment:
based on our discussion here:
https://seldondev.slack.com/archives/C8Y9A8G0Y/p1621264214141000?thread_ts=1620740550.073300&cid=C8Y9A8G0Y

This is not a Rest API, this is just an HTTP API. There is no resource in question such as a user or a document. We are not event doing a query here, such as querying for a document in Solr or ElasticSearch.

The appropriate distinction to make is from an actual `Rest API` and an `Inference API`. A `Rest API` would be an API to get or alter the state of the model - load, unload, update, get meta... etc. An `Inference API` would be this exact `/predict` endpoint which does some computation and spits out a result.